cmd/limactl/completion.go

```diff
@@ -19,64 +19,14 @@
 package main
```

Comment:
Now we can remove `// Forked from https://github.com/containerd/nerdctl/blob/v0.8.1/completion.go` in the first line